src/index.html

```diff
@@ -5,10 +5,60 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a class="logo">
+        <img
+        src="images/logo.png"
+        alt="logo"
+        >
+      </a>
+      <nav class="container">
+        <ul class="list">
+          <li class="item">
+            <a class="is-active link" href="#">Apple
+            </a>
+          </li>
+          <li class="item">
+            <a class="link" href="#">Samsung
+            </a>
+          </li>
```

Comment:
<img width="1122" alt="Spaces between tags" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/fa515810-e535-4895-ae9e-d49fa118a2d8">

Reply:
Fix in all places